packages/server-core/src/projects/project/project.hooks.ts

```diff
@@ -202,7 +202,7 @@ const ensurePushStatus = async (context: HookContext<ProjectService>) => {
       context.projectPushIds = context.projectPushIds.concat(matchingAllowedRepos.map((repo) => repo.id))
     }
 
-    if (!(await checkScope(context.params.user!, 'projects', 'read')))
+    if (!(await checkScope(context.params.user!, 'admin', 'admin')))
```

Comment:
why are we changing this from projects:read to admin:admin? don't we still want to use our scoped permissions? admin:admin should only be for critical infrastructure